examples/pipelines/controlnet-interior-design/pipeline.py

```diff
@@ -1,58 +1,38 @@
 """Pipeline used to create a stable diffusion dataset from a set of initial prompts."""
 # pylint: disable=import-error
-import json
+import logging
+import sys
 
-from kfp import components as comp
-from kfp import dsl
+sys.path.append("../")
 
-from config.general_config import KubeflowConfig
-from config.components_config import RetrieveImagesConfig
-from fondant.pipeline_utils import compile_and_upload_pipeline
+from pipeline_configs import PipelineConfigs
```

Comment:
Where is this imported from?

Reply:
It's here https://github.com/ml6team/fondant/blob/main/examples/pipelines/pipeline_configs.py at the root level, not sure why it does not show up on the github UI though

Reply:
Ok thanks, I was looking for a new file.